components/brave_new_tab_ui/components/default/settings/index.ts

```diff
@@ -452,3 +452,83 @@ export const ToggleCardsSwitch = styled<{}, 'div'>('div')`
   float: right;
   margin: -65px -10px 0 0;
 `
+
+export const StyledTopSitesCustomizationSettings = styled('div')`
+  --widget-gap: 10px;
+  font-family: ${p => p.theme.fontFamily.heading};
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  margin-top: 16px;
+  gap: var(--widget-gap)
+`
+
+export const StyledTopSitesCustomizationSettingsOption = styled<{}, 'div'>('div')`
```

Comment:
I don't think this is currently useable with a keyboard (or a screen reader) as it cannot have focus. The best for this is to use a `radio` (hiding the default circle appearance of the radio button. If that's too much work for now we could use 2 buttons?